### ISSUE TITLE
[Terminal >Set tab color]: Both label name and role is changing for 'More /Less' control.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12046]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Activate 'Actions' tab and then Add a new actions 'Set tab color' with shortcut key 'Ctrl + Shift + c'.
4. Open Narrator using 'Win + Ctrl + Enter' keys.
5. navigate inside Terminal window.
6. Press 'Ctrl + Shift + c' keys, then navigate to 'More/Less' control inside Set tab color pop-up using tab key.
7. Observe the issues.

**User Experience:**
Screen Reader users UX will not be good as they will get confused if control properties are not defined properly.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[Both label name and role is changing for 'More Less' control..zip](https://github.com/microsoft/terminal/files/7779898/Both.label.name.and.role.is.changing.for.More.Less.control.zip)

### Expected Behavior

For 'More' button Screen Reader should announce 'More button, invoke to show the text entry fields' and for 'Less' button Screen Reader should announce 'Less button, invoke to hide the text entry fields'

### Actual Behavior

Both label name and role is changing for 'More /Less' control.
i.e. For 'More' button Screen Reader is announcing 'More button Off, Invoke to show or hide the text entry fields' and For 'Less' button Screen Reader is announcing 'Less button On, Invoke to show or hide the text entry fields'

**Observation:** Issue repro with NVDA and JAWS Screen Readers.

**Comment by carlos-zamora** [comment]
Should be relatively easy. Sounds like a matter of defining help text and name to some controls in the tab color picker.

**Comment by carlos-zamora** [comment]
@v-rpundir this is a bug on WinUI 2: https://github.com/microsoft/microsoft-ui-xaml/issues/7395

**Comment by v-rpundir** [comment]
In Discussion with WinUI Testing team.

**Comment by v-rpundir** [comment]
Hi @carlos-zamora We are tracking this WinUI Bug here in their ADO https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/40517763 as WinUI team does not track issue in GitHub.

Thanks:):

**Comment by carlos-zamora** [comment]
Closing now that the bug has been accepted.
